test/integration/aens.ts

```diff
@@ -78,9 +79,9 @@ describe('Aens', function () {
   const pointers = {
```

Comment:
**line 50:** throws an `Returning an awaited promise is required in this context.` error. Should be  `retrun await...`